package.json

```diff
@@ -267,6 +267,7 @@
     "jsbi": "^3.1.4",
     "lightweight-charts": "^3.3.0",
     "make-plural": "^7.0.0",
+    "moment": "^2.29.4",
```

Comment:
Are we sure we want to add this library? 

Last time I check, moment was a bit heavy, and there was others like https://date-fns.org/ more light weight. 

Also, did you review if we have something already, I would swear for the token we showed relative dates

Reply:
indeed. It's important to make sure we don't increase the bundle size, and also we use the libraries we have already

timeago.js is a dependency

https://pub.dev/packages/timeago

we even have a hook https://github.com/cowprotocol/cowswap/blob/d254296019a6841883a2a459a936c3e25f131f53/src/custom/hooks/useTimeAgo.ts#L4

Reply:
Replaced moment.js with `date-fns` :+1: